### ISSUE TITLE
Bug: worklogs not being submitted properly sometimes

### ISSUE THREAD

**Issue by Gazotey** [tulleuchen/jirastopwatch#161]
There's this bug where worklogs are not showing up in Jira after being submitted in Jira Stopwatch.
Note that this issue shows up rather sporadically.

- Log time on a ticket. (normal behaviour)
- Click the button to submit the worklog. Popup opens. (normal behaviour)
- Don't fill in a comment (normal behaviour)
- Press _Submit_. Popup closes and checkbox of ticket resets to _0m_ (normal behaviour)
- Check in Jira _Tempo_. The worklog is missing. (unexpected behaviour)

When the worklog is missing, it's really gone. It doesn't show up in Tempo (even after a few days) and the ticket timer in _Jira Stopwatch_ is reset after submitting. This issue shows up every now and then, but rarely more than once a week.

I appreciate this tool very much and - by extension - the owner and maintainers as well. So thank you for your efforts!
By the way, are contributions to this repository still appreciated? I see recent activity on the repo, but also stale PR's from early 2018. If so, I'd love to attempt to fix this bug.

**Comment by vbjay** [comment]
Were you connected at the time?

**Comment by Gazotey** [comment]
@vbjay Yes, I was. At the end of the day, I always have at least a few tickets that I logged time on. I submit all of the worklogs, check in Tempo, and - every now and then - see that one is missing. The other ones were submitted just fine.
